### DISCUSSION TITLE
执行docker-compose up -d报错

### DISCUSSION THREAD

**Discussion by ChristopherSung** [MoeNetwork/Tieba-Cloud-Sign#207] [疑问]
报错内容
yaml: line 28: mapping values are not allowed in this context

**Comment by n0099** [comment]
哪个yaml文件错了

**Reply by ChristopherSung** [reply]
docker-compose.yml 文件

**Reply by n0099** [reply]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/blob/023316e0df05ddafd55060717f399d617854c618/docker/docker-compose.yml#L28-L29

**Reply by n0099** [reply]
整个yaml文件是合法的啊

**Reply by ChristopherSung** [reply]
感谢回复，我再重新装一下吧
